hips/draw/ui.py

```diff
@@ -116,10 +116,10 @@ def write_image(self, filename: str, overwrite: bool = False) -> None:
 
         if self.tile_format == 'fits':
             hdu = fits.PrimaryHDU(data=self.image, header=self.geometry.fits_header)
-            hdu.writeto(filename)
+            hdu.writeto(filename, overwrite=overwrite)
         else:
             image = Image.fromarray(self.image)
-            image.save(filename)
+            image.save(filename,j, overwrite=overwrite)
```

Comment:
```suggestion
            image.save(filename, overwrite=overwrite)
```
I think you added the extra `j` by mistake. Otherwise, this looks good to me.